common/src/chain/config/mod.rs

```diff
@@ -153,6 +157,26 @@ impl ChainConfig {
         self.max_block_size_with_smart_contracts
     }
 
+    pub fn token_min_issuance_fee(&self) -> Amount {
+        self.token_min_issuance_fee
+    }
+
+    pub fn token_max_uri_len(&self) -> usize {
+        self.token_max_uri_len
+    }
+
+    pub fn token_max_dec_count(&self) -> u8 {
+        self.token_max_dec_count
+    }
+
+    pub fn token_max_ticker_len(&self) -> usize {
+        self.token_nax_ticker_len
```

Comment:
nax...

Reply:
I've fixed this and a load of other English issues. I'll push it when I'm done

Reply:
My bad, another misprint